common/lib/protocol-definitions/package.json

```diff
@@ -60,7 +60,7 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "1.1.0",
+    "version": "1.2.0",
```

Comment:
This seems like an unrelated change. is it related somehow?

Reply:
I think that when the version is umped the typeValidation version should also be bumped. That said, it's inconsistent to do it in this PR -- I'll remove this and come back through with typetest updates as needed.